packages/default-field-editors/package.json

```diff
@@ -42,6 +42,7 @@
     "@contentful/field-editor-tags": "^1.0.1",
     "@contentful/field-editor-url": "^1.0.1",
     "@contentful/field-editor-validation-errors": "^1.0.1",
+    "contentful-management": "^7.45.2",
```

Comment:
Can we increase the version range to maybe `^7.22.0` when the defaults were introduced?

Reply:
Good point. Done